packages/idd/idd.0.1/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "2.0"
+name: "idd"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build}
+  "base"
```

Comment:
I think you need a lower bound on v1.12 o v1.13. There have been various breaking changes in the releases of base

Reply:
Why v1.12 or v1.13 if v1.0 hasn't been released yet?

Reply:
Sorry, ot was a typo, I meant 0.